libs/blog/shell/feature-shell-web/src/lib/blog-shell.routes.ts

```diff
@@ -21,13 +21,19 @@ export const blogShellRoutes: Route[] = [
         loadComponent: async () =>
           (await import('@angular-love/feature-about-us'))
             .FeatureAboutUsComponent,
+        data: {
+          seoTitle: 'About Us',
+        },
```

Comment:
all seo parameters should be wrapped within some object like "seo", to aggregate all seo-related stuff into one container
```suggestion
        data: {
          seo: {
            title: "About Us"
          }
        },
```